ambuild2/frontend/v2_1/cpp/detect.py

```diff
@@ -95,7 +95,7 @@ def TryVerifyCompiler(env, mode, cmd, assumed_family):
       util.ConsoleHeader,
       'Compiler {0} for {1} failed: '.format(cmd, mode),
       util.ConsoleRed,
-      e.message,
+      str(e),
```

Comment:
This should be a separate PR, with an explanation (also probably just changing it to `e` is fine).